GPU/GPUCommon.cpp

```diff
@@ -1543,6 +1543,9 @@ void GPUCommon::Execute_Prim(u32 op, u32 diff) {
 	// PRIM commands with other commands. A special case that might be interesting is that game
 	// that changes culling mode between each prim, we could just change the triangle winding
 	// right here to still be able to join draw calls.
+	if (debugRecording_)
```

Comment:
Might be nice to add ` || host->GPUDebuggingActive()` here so that step prim works.

-[Unknown]

Reply:
I guess that depends on what you're debugging, it's kind of nice when stepping prim to see that the inner interpreter succeeds in running them in one pass, plus we don't flush after each prim when it does anyway so there's not much to see. But yeah, I guess in the usual case it's confusing to have it jump like that..